docs/aries-interop-intro.md

```diff
@@ -19,69 +19,87 @@ What unites Aries agents are the standards and protocols they aim to adhere to,
+
+#### Investigating Failing Tests
+
+The Allure reports accessible from this site provide a lot of information about failing tests and are a good place to start in figuring out what is happening. Here's how to follow the links to get to the test failure details:
+
+1. On the main page, pick a Test Agent whose tests you want to drill into by clicking the name of the Test Agent from the main page summary table (first column), taking you to the Test Agent page.
+2. On the Test Agent page, find a runset with at least some failures, and click on the runset name from the Test Agent summary table (first column), taking you to the section of the Test Agent page about that runset.
+3. Within the runset details section, check the "Current Runset Status" summary to see if there is any reason for the failures there, and then drill into the test results by clicking the link entitled "Results by executed Aries RFCs." Clicking that link takes you from the Aries Interop Info site into the Allure test results, and lots (and lots) of details about test runs.
+4. The page you will arrive upon will show the handful of RFCs scenarios executed during the test runs (with titles like "RFC 0036 Aries agent issue credential"), and for each a count of the status for tests cases within each scenario (e.g. passed, failed, broken and so on).
+5. Expand a scenario with one or more failing tests and then click on the one of the failed tests (with an ugly red "X" beside the test case) to see the details (stack trace) of the error on the right part of the page.
+6. From here you can look at a variety of things about the failing test:
+   1. Review the test failure "Overview" to see the specific test step that failed and related failed assertion stack trace.
+   2. Scroll down to the sequence of steps in the test, to see how far along the test got before the failure.
+   3. Click on the "History" tab to see if it has always failed, or if this is a new issue.
+   4. If it has passed in earlier runs, drill into the passing test to see if you can learn anything from that.
+   5. Dive into the weeds...look at the stack trace, find the associated code, and see if you can figure out what happened. Find anything?  Report it via an issue, or even better, submit a Pull Request to fix the issue. Remember to consider all of the possible places an error could occur (above).
+
+In addition to drilling into a specific test scenario (aka "stories")/case (aka "behavior")/step, you can look at the entire runset history. On the left side menu, click on "Overview", and then take a look at the big "history" graph in the top right, showing how the runset execution has varied over time. Ideally, it's all green, but since you started from a runset that had failures, it won't be. Pretty much every part of the overview page is a drill down link into more and more detailed information about the runset, a specific run of the runset, a specific test case and so on. Lots to look at!
```

Comment:
"entire runset history" is not actually the case. It is capped at 20 executions. This was to manage space on the Allure server. We can increase that. We thought 20 was good enough to show some recent history, and we probably don't care about any older than that.